html/ui/js/brs.settings.js

```diff
@@ -2,6 +2,8 @@
  * @depends {brs.js}
  */
 var BRS = (function(BRS, $, undefined) {
+	var API_URL = "https://explorer.burstcoin.network/api_activate.php";
```

Comment:
This is my main problem with this, it adds a dependency on an external service, which I don't consider appropriate for BRS. I wouldn't feel the same way about Phoenix, but I think BRS must remain fully independent

Reply:
I tend to agree. Further, we should feature freeze this UI, as in no new features. Closing this.